src/jit/target.h

```diff
@@ -469,7 +469,7 @@ typedef unsigned char   regNumberSmall;
   // See vm\i386\asmhelpers.asm for more details.
   #define RBM_PROFILER_ENTER_TRASH     RBM_NONE
   #define RBM_PROFILER_LEAVE_TRASH     RBM_NONE
-  #define RBM_PROFILER_TAILCALL_TRASH  (RBM_ALLINT & ~RBM_ARG_REGS)
+  #define RBM_PROFILER_TAILCALL_TRASH  (RBM_CALLEE_TRASH & ~RBM_ARG_REGS)
```

Comment:
If I remember correctly `CALEE_TRASH = CALEE_INT_TRASHED + CALEE_FLOAT_TRASHED`, do you want to use only  `CALEE_INT_TRASHED` here or do you need both?

Reply:
Well, `RBM_CALLEE_TRASH` is correct. At least, for our implementation of `ProfileTailcallNaked` helper on x86.

But you made me thinking if different helper - `ProfileEnterNaked` on x86 can accidentally trash floating argument registers. I would postpone this discussion for the next step - I was planning to update Noah's PR #19560 and will verify those.